internal/store/files/WorldGeosForPlaceRecognition.csv

```diff
@@ -104041,3 +104041,73 @@ undata-geo/G00800370,GeoRegion,"Western Asia (excl. Armenia Azerbaijan Cyprus Is
+bio/Betapapillomavirus1_X74471,VirusIsolate,"Betapapillomavirus 1 X74471",,
+bio/Betapapillomavirus1,Virus,"Betapapillomavirus 1",,
+bio/BetainfluenzavirusInfluenzae,Virus,"Betainfluenzavirus influenzae",,
+bio/AlphainfluenzavirusInfluenzae,Virus,"Alphainfluenzavirus influenzae",,
+bio/Betacoronavirus1,Virus,"Betacoronavirus 1",,
+bio/HumanOrthopneumovirus,Virus,"Human orthopneumovirus",,
+bio/ACAPO,Species,AcanthamoebaPolyphaga,,
+bio/9ACTN,Species,Actinobacteria,,
+bio/9ENTO,Species,Enterovirus,,
+bio/9CYAN,Species,Cyanobacteria,,
+bio/9FUNG,Species,Fungi,,
+dc/001zpbbtbyqm3,BiologicalSpecimen,"Cyperus strigosus L.",,
+dc/003ye83smktt7,BiologicalSpecimen,Volvox,,
+dc/004b8q2c1sc6b,BiologicalSpecimen,"Achillea millefolium L.",,
+dc/004pn4gc3h683,BiologicalSpecimen,"Corylus cornuta Marshall",,
+dc/002q6x1nl6nxh,BiologicalSpecimen,"Euphorbia heterophylla L.",,
+bio/rs13317,GeneticVariant,rs13317,,
+bio/rs7903146,GeneticVariant,rs7903146,,
+bio/rs2237897,GeneticVariant,rs2237897,,
+bio/rs72636191,GeneticVariant,rs72636191,,
+bio/rs559628884,GeneticVariant,rs559628884,,
+bio/hg38_FGFR1,Gene,FGFR1,,
+bio/hg38_APOE,Gene,APOE,,
+bio/ACHE,Gene,ACHE,,
+bio/ABRAXAS2,Gene,ABRAXAS2,,
+bio/AADAT,Gene,AADAT,,
+bio/DOID_0080176,Disease,"meningococcal meningitis",,
+bio/DOID_0050204,Disease,"Epstein-Barr virus hepatitis",,
+Chickenpox,Disease,Chickenpox,,
+Chlamydia,Disease,Chlamydia,,
+bio/DOID_0050052,Disease,"Rocky Mountain spotted fever",,
+ICD10/A30-A49,ICD10Section,"Other bacterial diseases (A30-A49)",,
+ICD10/A00-B99,ICD10Section,"A00-B99 (Certain infectious and parasitic diseases)",,
+ICD10/J00-J06,ICD10Section,"Acute upper respiratory infections (J00-J06)",,
+ICD10/B95-B97,ICD10Section,"Bacterial and viral infectious agents (B95-B97)",,
+ICD10/I70-I79,ICD10Section,"Diseases of arteries, arterioles and capillaries (I70-I79)",,
+ICD10/A39,ICD10Code,"Meningococcal infection",,
+ICD10/R97,ICD10Code,"Abnormal tumor markers (R97)",,
+ICD10/A39.2,ICD10Code,"Acute meningococcemia",,
+ICD10/Z68,ICD10Code,"Body mass index [BMI] (Z68)",,
+ICD10/A00,ICD10Code,Cholera,,
+bio/D001172,MeSHDescriptor,"Arthritis, Rheumatoid",,
+bio/D011655,MeSHDescriptor,"Pulmonary Embolism",,
+bio/D004110,MeSHDescriptor,Diltiazem,,
+bio/D000066888,MeSHDescriptor,"Diet, Food, and Nutrition",,
+bio/D000066449,MeSHDescriptor,"Human Embryonic Stem Cells",,
+chem/CID1983,Drug,acetaminophen,,
+chem/CID681,Drug,dopamine,,
+bio/Bactocill,Drug,Bactocill,,
+bio/Aggrenox,Drug,Aggrenox,,
+chem/CID5734,Drug,zonisamide,,
+chem/A03FA10,AnatomicalTherapeuticChemicalCode,acotiamide,,
+chem/A01AC03,AnatomicalTherapeuticChemicalCode,hydrocortisone,,
+chem/A02AC01,AnatomicalTherapeuticChemicalCode,"calcium carbonate",,
+chem/A01AB22,AnatomicalTherapeuticChemicalCode,doxycycline,,
+chem/A06AG04,AnatomicalTherapeuticChemicalCode,glycerol,,
+bio/C523297,MeSHSupplementaryRecord,"N,N-didesmethyldiltiazem",,
+bio/C000128,MeSHSupplementaryRecord,rubiflavin,,
```

Comment:
Could you please add it to a different file and concat on load?

Reply:
done